docs/TESTS.md

```diff
@@ -12,7 +12,7 @@ Your registry module repository will need to be a [valid module](https://www.ter
 It will need the following: 
 1. To be named `terraform-<PROVIDER>-<NAME>`
 1. At least one valid SemVer tag in the format `x.y.z`
-[terraform-random-module](ttps://github.com/caseylang/terraform-random-module) is a good example repo. 
+[terraform-random-module](https://github.com/caseylang/terraform-random-module) is a good example repo. 
```

Comment:
small little typo fix that I didn't think was worth a second PR, so I snuck it in here 🥷